avida-ts/README.md

```diff
@@ -0,0 +1,37 @@
+# Typescript library for AVIDA
```

Comment:
Add building contract and schemas stages to doc

Reply:
added doc in https://github.com/nymlab/AVIDA/pull/6/commits/7ce3c1886296d88d19edcfd650036b81c8dcdf75